webserver/website/src/App.vue

```diff
@@ -1,86 +1,28 @@
-            class="navbar-toggler"
-            type="button"
-            data-bs-toggle="collapse"
-            data-bs-target="#navbarSupportedContent"
-            aria-controls="navbarSupportedContent"
-            aria-expanded="false"
-            aria-label="Toggle navigation"
-          >
-            <span class="navbar-toggler-icon"></span>
-          </button>
-          <div class="collapse navbar-collapse" id="navbarSupportedContent">
-            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-              <li class="nav-item">
-                <router-link class="nav-link" to="/">Verbindungssuche</router-link>
-              </li>
-              <li class="nav-item">
-                <router-link class="nav-link" :to="{ path: '/about', hash: '#content' }">Über TCP</router-link>
-              </li>
-              <li class="nav-item dropdown">
-                <a class="nav-link dropdown-toggle" data-toggle="dropdown">Daten</a>
-                <ul class="dropdown-menu">
-                  <li>
-                    <router-link class="dropdown-item" :to="{ path: '/data/stats', hash: '#content' }">Übersicht</router-link>
-                  </li>
-                  <li>
-                    <router-link class="dropdown-item" :to="{ path: '/data/stations', hash: '#content' }"
-                      >Stationen</router-link
-                    >
-                  </li>
-                  <!-- <li>
-                  <router-link class="dropdown-item" :to="{ path: '/data/obstacles', hash: '#stats' }"
-                    >Zug-Hindernisse</router-link
-                  >
-                </li> -->
-                </ul>
-              </li>
-            </ul>
-            <div class="d-flex">
-              <a
-                class="pretty_button"
-                href="https://github.com/TrAInConnectionPrediction/tcp"
-                target="_blank"
-                rel="noopener"
-                ><i class="tcp-github"></i>TCP auf GitHub</a
-              >
-            </div>
-          </div>
-        </div>
-      </nav>
-      <div id="pgr_bar"></div>
-    </div>
-    <div id="intro" class="view">
-      <div class="d-flex justify-content-center align-items-center mask" style="height: 100%">
-        <div class="row" style="justify-content: center; min-width: 0; width: 100%">
-          <div class="col white-text text-center">
-            <h2 id="midheader" class="shadowheader">TrAIn_Connection_Prediction: TCP<br /></h2>
-            <hr class="hr-light" />
+    <navbar></navbar>
+    <div id="intro" class="container-md">
+      <div class="d-flex justify-content-center align-items-center" style="height: 100%">
+        <div class="hero-layout">
+            <div class="slogan d-flex flex-column justify-content-center align-items-center">
+            <h2>Nächstes mal pünktlich</h2>
             <p>
-              <strong> Ihre Verbindungsvorhersage </strong>
+              <strong>Dank unserem Verbindungsranking</strong>
```

Comment:
Ich finde des passt irgendwie nicht.
Da muss entweder ein Logo hin, unser Name, oder der Text muss ganz weg.
Weil "Nächstes mal..." ist zwar ein netter Slogan und so, aber der kriegt da auf der Landing Page als einziger Text viel zu viel attention.

Wenn man da nen Logo drüber klatscht, wie wir des davor hatten, glaube ich geht des wieder, aber alleine finde ichs nicht so gut.

Ganz extrem -- von mir aus könnte man auch des design der Landing page etwas ändern?

Reply:
Ja 100% zufrieden bin ich damit auch nicht. Es war auch der ersten "Slogan" der mir eingefallen ist. Ich würde mich halt bald mal mit dem einen Typen da treffen und mir mit ihm zusammen Gedanken über den Namen machen und dann kann man den da wieder einfügen. Ich hatte auch zwischendurch mal damit experimentiert, einfach nur das Form dazuhaben, aber dafür hatte das Form finde ich noch zu wenig "content".
Und nen wirkliches Logo haben wir halt nicht. Wenn du da nen Guten vorschlag hast, können wir es auf jeden Fall sofort ändern. Die Frage ist: ist es jetzt schlechter als davor.

Reply:
Bis wir einen neuen slogan oder so haben brauchen sollte irgendwo auf der Webseite TrAIn_Connection_Prediciton stehen 
Von mir aus auch nur oben in der Topbar, aber irgendwo sollte es schon noch drinnen sein

Reply:
Wie findest du das jetzt?

Reply:
ist fürs erste ok